### ISSUE TITLE
Getting JS Certificate

### ISSUE THREAD

**Issue by IMadeTheDevilRun** [freeCodeCamp/freeCodeCamp#54895]
I wanted to know, for getting JS certificate which is on the second part of the course, should I already passed the html/css course or not?! 
I mean is it possible to start from the JS section and get its certificate by passing its 5 challenges without knowing html or css?!!

**Comment by zxc-w** [comment]
Yes, you can get the JavaScript certificate without completing HTML/CSS course or getting its certificate.

That's only if you already have prior knowledge of HTML/CSS and want to start with JavaScript straightforward.
